modules/virt-about-runstrategies-vms.adoc

```diff
@@ -4,28 +4,52 @@
 
```

Comment:
I believe the assembly list at the top of the file should now be updated since the module was moved. From:
```
// * virt/virtual_machines/virt-create-vms.adoc
```
to:
```
// * virt/node_maintenance/virt-about-node-maintenance.adoc
```